apps/src/sites/studio/pages/layouts/application.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery';
+
+window.jQuery = window.$ = $;
+
+require('jquery-ujs');
+require('jquery-ui');
+
+require('bootstrap');
+require('jquery-mask-plugin');
+require('jquery-placeholder');
+require('qtip2');
+require('jquery-query-object');
+require('timeago');
+require('selectize');
+
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/add2home.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/jquery.simulate.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/jquery.ui.touch-punch.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/jquery.overlaps.js');
+require('script-loader!@cdo/dashboard/vendor/assets/javascripts/md5.js');
+require('script-loader!@cdo/shared/js/details-polyfill/jquery.details.js');
+require('script-loader!@cdo/shared/js/details-polyfill/details-polyfill.js');
```

Comment:
We may want to use `script-loader` for all of these libraries and not just the vendor ones. I think my strategy would be to copy the files in question from the ruby gems directly into the vendor directory and then just ditch the ruby gems. The would probably get us the closest to version parity with the whole application.js.erb file, but it also feels quite ugly to me. Curious about what you guys think about this!